src/main/java/org/wecancodeit/peaceofmind/Phone.java

```diff
@@ -0,0 +1,23 @@
+package org.wecancodeit.peaceofmind;
+
+public class Phone {
+
+	private String phoneNumber;
+	private String phoneType;
```

Comment:
let's just change this to 'type' as that's what we're hoping to enforce with @Michael-Mosher 's type interface :)

Reply:
Changed phoneType to simply to type on both field and method.